iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerNoScan.java

```diff
@@ -587,6 +618,7 @@ public void testIcebergAndHmsTableProperties() throws Exception {
     expectedIcebergProperties.put("custom_property", "initial_val");
     expectedIcebergProperties.put("EXTERNAL", "TRUE");
     expectedIcebergProperties.put("storage_handler", HiveIcebergStorageHandler.class.getName());
+    expectedIcebergProperties.put(serdeConstants.SERIALIZATION_FORMAT, "1");
```

Comment:
Why is this change?

Reply:
In this PR, I also added a new logic to the `IcebergMetaHook` to copy the serdeproperties from HMS table to the catalog properties. This change was required since the hive syntax allows to provide additional serdeproperties when creating non-native tables. Like:
`CREATE TABLE .... STORED BY ICEBERG WITH SERDEPROPERTIES('my_key'='my_value')`

When we create a table, hive automatically updates the serdeproperties with the `SERIALIZATION_FORMAT` set to `1`( if it's not overwritten from the DDL).

Reply:
Thanks for the explanation!